contexts/citizenship-v1.jsonld

```diff
@@ -37,18 +40,33 @@
         "schema": "http://schema.org/",
         "xsd": "http://www.w3.org/2001/XMLSchema#",
```

Comment:
Since we must put out a new version of this context anyway, we should remove all the prefixes and use full URLs.

Reply:
Yes, I'm concerned these changes are being commit to the v1 context. This needs to be a v2 context